### ISSUE TITLE
Gründung: Schirmherrschaft / Verein

### ISSUE THREAD

**Issue by acidicX** [ODL-Augsburg/orga#1]
Wie bekommen wir das ODL auf die Beine?

Am schnellsten wäre sicherlich, wenn ein bestehende "juristische Person" die Schirmherrschaft übernimmt. Eine eigene Vereinsgründung wäre deutlich umfangreicher und mit jährlichem Aufwand verbunden.

Ich glaube, dass diese Entscheidung zum Großteil davon abhängen wird, ob wir einen Partner/Sponsor für die Räumlichkeiten finden. Wenn die Organisation für den Raum keine Miete zahlen muss, dann ist das weitere Risiko abschätzbar und wohl auch durch eine Privatperson tragbar.

**Comment by MartinKolb1987** [comment]
ich hab letztens bei team23 auch etwas über ein odl gelesen --> https://www.facebook.com/permalink.php?id=143977845671692&story_fbid=905612946174841 --> kannst dich ja mal mit denen in Verbindung setzen. Wie weit die sind etc...

**Comment by meaku** [comment]
Cool! Wir sollten definitiv mit Team23 und dem Stefan Schimpfle vom aiti Park reden.

**Comment by acidicX** [comment]
Jep! Das führt natürlich auch zu der Frage, unter welchen Umständen ein Raum gestellt werden könnte.

Selbst wenn der Raum in der Nähe eines Unternehmens ist, sollte er doch deutlich getrennt von deren Büros sein, oder? Sonst müsste man immer koordinieren, damit man nicht gerade in ein Meeting oder einen Kundentermin platzt... das killt spontane Besuche und lässt das "Open" im Name auch etwas schlecht aussehen ;-)

**Comment by MartinKolb1987** [comment]
Klar, aber wir wissen ja nicht wie Team23 das löst ;-). Einfach mal nachfragen.

**Comment by acidicX** [comment]
Jop, ich hab gestern schon eine Mail geschrieben - bisher keine AW.

**Comment by acidicX** [comment]
Ich hab mich jetzt etwas länger mit Fabian von T23 ausgetauscht: es ist so, wie ich es vermutet habe. Der Raum ist bei ihnen im Büro, daher wird man als "Besucher" zwangsläufig immer Termine machen müssen. Auch ein Ausleihen der Geräte ist erstmal nicht angedacht. Finde ich jetzt persönlich eher unbefriedigend, aber es stellt sich die Frage, ob man eine bessere Alternative organisieren kann. Sie wollen auf jeden Fall ihr ODL auch im Rahmen des nächsten Web&Wine mal vorstellen.

Mir schwebt da so etwas vor wie ein offener Raum (Schließanlage ähnlich wie an der FH, von X bis X Uhr offen mit Token/PIN), in dem einfach ein Spindschrank mit Zahlenschlössern steht (wie an der FH).

@meaku wollen wir mal einen Termin bei Stefan Schimpfle machen?

**Comment by meaku** [comment]
@acidicX merci fürs auschecken. Ich versuch mal den Stefan aufm Gang zu erwischen und dann ggf. nen Termin zu machen.
